Add example with cookies-next package (#31243)
vercel/next.js: examples/with-cookies-next/README.md

# With cookies-next

In this example, we are getting, setting and removing cookies

This package works on client side and server side

- can be used on the client side, anywhere
- can be used for server side rendering in getServerSideProps
- can be used in API handlers

> ## Preview
> 
> Preview the example live on [StackBlitz](http://stackblitz.com/):
> 
> [![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/vercel/next.js/tree/canary/examples/with-cookies-next)
> 
> 
> ## Deploy your own
> 
> Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example):

Reviewer: ```suggestion
## Deploy your own

Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example) or preview live with [StackBlitz](https://stackblitz.com/github/vercel/next.js/tree/canary/examples/with-cookies-next)
```

Let's make this consistent with other examples
